script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -356,97 +357,99 @@ The scattered plates. The scattered bodies.
 }
 [sha:2ffa312d27ee4f1d94937373e6c0b3203aa66f44]{
 -- Page 597, Offset 34975.
 -- 街は二つの人種に分かれていた。
-The town is separated into two types of people.
+The town has been separated into two types of people.
 }
 [sha:f2a0f5ea7fffdd78942c6a7d3dc7a2b66265f90f]{
 -- Page 597, Offset 34976.
 -- 追う人と逃げる人。殺す側と殺される側。
-Those who chase, and those who flee. The side that kills, and the side that is killed.
+Those who pursue, and those who flee. Those who kill, and those who are killed.
 }
 [sha:60b5bff736966a96a8c1ddd20b4d89b5a310bf90]{
 -- Page 597, Offset 34977.
 -- さっきまで人間だった生き物と、
-The creatures that were human a moment ago,
+The creatures that were human just moments ago―――
 }
 [sha:1b2003e956f0f34c668639226e117f0cb55be266]{
 -- Page 597, Offset 34978.
 -- まだ人間として逃げる生き物と。
-and the creatures that are still human and run away.
+and the creatures that are still human and running away.
 }
 [sha:f6c8540c814f0089b9bb984d2edeb8e7f4bc70e7]{
 -- Page 598, Offset 34979.
 -- 何がどうしてこうなったのかは解らない。きっと一生解らない。ただ、わたしがベッドでいじけている間に世界終了のお知らせがあって、わたしだけがそれを聞き逃してしまったから、お父さんとお母さんに置き去りにされてしまったんだ、と悲しくなった。
-I don't know why any of this is happening. I'll probably never know for the rest of my life. All I know is that while I was being down in my own room, the world was coming to an end, and because I was the only one who missed the warning, my mother and father ended up leaving me behind. A fact that makes me sad.
+I don't know why any of this is happening. I'll probably never know for the rest of my life. All I know is that while I was moping in bed, there was an alert that the world was going to end, and because I was the only one that missed it, my parents abandoned me. Thinking about that fact fills me with sorrow.
 }
 [sha:adffa983ddd9d18edd70c4090ccb9a18206eab2f]{
 -- Page 599, Offset 34980.
 -- 悲鳴をあげられないのが幸いしたのか、わたしは怪物たちに見つからずにメインストリートから抜け出せた。
-I manage to slip out of the main street without being spotted by the monsters, perhaps because I was fortunate enough to not raise a scream.
+Because I didn't even have the chance to scream, I manage to slip through the main street without being noticed by the monsters.
 }
 [sha:88eefb81201021471e6c2daf01ab9c66b435f6fa]{
 -- Page 599, Offset 34981.
 -- とにかくお父さんたちに会いたかった。
-For the time being, I just wanted to see my father and everyone else.
+I want nothing more right now than to see my parents.
 }
 [sha:b5f9c8c34f678245920900068fb10454a8ab756a]{
 -- Page 599, Offset 34982.
 -- ここは危険だから今ごろは街の外に出てわたしを待っているに違いない。
-Since this place is dangerous, I'm sure they went out of the city and are waiting for me right now.
+It's no longer safe here, so they must be waiting for me outisde of town.
 }
 [sha:5bd5783f3f4cd7cf38cb75c8cc2f4e99fb84f8d4]{
 -- Page 600, Offset 34983.
 -- メインストリートから二番街に抜けて、坂道を下りきれば国道に出られる。
-If I go from the main street to past the second block and down the hill, I can reach the main road.
+// i have absolutely no idea how all this road lingo works irl so idk if this even makes sense
```

Comment:
pfft, idk. leave it for the boys in dover looks fine to me

Reply:
my man 
![image](https://user-images.githubusercontent.com/97076550/203936324-36d75ab9-683e-4e2a-9648-054bd7f02d4d.png)